schematic/store/synapse.py

```diff
@@ -1119,6 +1119,13 @@ def _get_file_entityIds(
                     "No manifest was passed in, a manifest is required when `only_new_files` is True."
                 )
 
+            if "entityId" not in manifest.columns:
+                raise ValueError(
+                    "The manifest in your dataset and/or top level folder must contain the 'entityId' column. "
+                    "Please generate an empty manifest without filling in the annotations, "
+                    "add files to your manifest and try again."
```

Comment:
I think having the last part of the message say `Please generate an empty manifest without annotations, manually add annotations to the appropriate files in the manifest, and then try again.` Would be a little more clear since when they regenerate the manifest the files and entityIds will be added automatically but otherwise it looks good!

Reply:
I think that is ok. I got feedback from this slack thread: https://sagebionetworks.slack.com/archives/C01ANC02U59/p1732144795107819?thread_ts=1732144795.107819&cid=C01ANC02U59.

But I do think what you have stated is more clear.

Reply:
@BWMac I think the text is incorrect but we can wait for final feedback

Reply:
>@GiaJordan do you think that we should even suggest that they can fill out entityId themselves? For example, super users who know what they're doing may want to fill in the entityId column without having to fill out an entirely blank sheet again. I know that this is what the HTAN community has been doing instead of generating an empty manifest.

@thomasyu888 we could. The risk is when files/entities(folders) are annotated it's based on the entityId. If there was a mismatch then the wrong annotations would be added to the files.

Reply:
Thanks @GiaJordan ^ That's true, should we include that in the error message like:

```
The manifest in your dataset and/or top level folder must contain the 'entityId' column. Please generate an empty manifest without annotations, manually add annotations to the appropriate files in the manifest, and then try again. For super users, you may fill out entityId's yourself, but you risk adding annotations to the wrong File entity if you fill out the wrong entityId"